Makefile

```diff
@@ -108,6 +110,31 @@ tag: ## Set a docker tag to the image.
 push: ## Push docker image.
 	docker push $(IMAGE_PREFIX)pvc-autoresizer:$(IMAGE_TAG)
 
+##@ Chart Testing
+
+.PHONY: ct-lint
+ct-lint: ## Lint and validate a chart.
+	$(SUDO) docker run \
```

Comment:
Could you explain why we need sudo? I think we can assume users are in the docker group.